src/containers/LearningResourcePage/components/LearningResourcePanels.jsx

```diff
@@ -46,6 +47,13 @@ const panels = [
     errorFields: ['metaDescription', 'metaImageAlt', 'tags'],
     component: props => <FormikMetadata {...props} />,
   },
+  {
+    id: 'learning-resource-competences',
+    title: 'form.competences.label',
+    className: 'u-6/6',
+    errorFields: ['competences'],
```

Comment:
Her sier du at `competences` er et felt som skal valideres, men ser ikke at du har oppdatert valideringsreglene noe sted?
Er det intentional?

Reply:
Legger til validation på Competences as we speak